apollo-runtime/src/main/java/com/apollographql/apollo/fetcher/ApolloResponseFetcher.java

```diff
@@ -0,0 +1,47 @@
+package com.apollographql.apollo.fetcher;
+
+import com.apollographql.apollo.internal.fetcher.CacheFirstFetcher;
+import com.apollographql.apollo.internal.fetcher.CacheOnlyFetcher;
+import com.apollographql.apollo.internal.fetcher.CacheAndNetworkFetcher;
+import com.apollographql.apollo.internal.fetcher.NetworkFirstFetcher;
+import com.apollographql.apollo.internal.fetcher.NetworkOnlyFetcher;
+
+public final class ApolloResponseFetcher {
```

Comment:
nit: should be named `ApolloResponseFetchers`, like in java `Executors`, in Rx `Schedulers`